docs/develop/07_dApp/02_well-known-did-config.md

```diff
@@ -106,5 +106,5 @@ How the file is hosted depends on your project setup and is out of scope for thi
 [concept-attestations]: ../../concepts/05_credentials/04_attestation.md
 [cookbook-attestations]: ../01_sdk/02_cookbook/04_claiming/03_attestation_creation.md
 [create-full-did]: ../01_sdk/02_cookbook/01_dids/02_full_did_creation.md
-[well-known-spec]: https://identity.foundation/.well-known/resources/did-configuration/
+[well-known-spec]: https://specs.identity.foundation/specs/did-configuration/
```

Comment:
The `specs.identity.foundation` doesn't work for me. 🤔 
Without `specs.` it works.

```suggestion
[well-known-spec]: https://identity.foundation/specs/did-configuration/
```